build-tools/build-sets

```diff
@@ -5,12 +5,12 @@ const fs = require("fs");
 const child_process = require('child_process');
 const path = require("path");
 
-process.stdout.write("Importing sets from @pokemon-showdown/sets... ");
+process.stdout.write("Importing sets from @Showdown-Summit-Server/sets... ");
```

Comment:
Excuse me - what? Please do not change the package for the sets here.

Reply:
also forgot to change